_source/user-guide/distributed-tracing/k8s-deployment.md

```diff
@@ -22,6 +23,7 @@ _Before you begin:_
 
 Step 1.
```

Comment:
@YajB Do you have a convention for the numbering system?

Reply:
Yes. 
I will use markdown, so that the steps are formatted with numbers in circles, as here: https://docs.logz.io/user-guide/distributed-tracing/getting-started-tracing/